test/spectcl.js

```diff
@@ -348,16 +348,35 @@ describe('spectcl', function(){
         })
     })
 
+    describe('send', function(){
+        it('should call the cb once data has been written to child', function(done){
+            var session = new Spectcl()
+            session.spawn('node --interactive')
+            session.expect([
+                '>', function(){
+                    assert.notEqual(session.expect_out.match, null, 'null expect_out match')
+                    assert.notEqual(session.expect_out.buffer, '', 'empty expect_out buffer')
+                    session.send('process.exit()\r', function(){
+                        session.child.on('exit', function(){
+                            done()
+                        })
+                    })
+                }
+            ])
+        })
+    })
+
     describe('sendEof', function(){
         it('should kill the child process, emitting an exit event', function(done){
             var session = new Spectcl()
-            session.on('exit', function(){
```

Comment:
I'd also update this test to reflect the changes to `child.on('close',...`, so that you catch the session `exit` event and the cb to `sendEof`.